Dockerfile

```diff
@@ -27,18 +27,20 @@ ENV LD_PRELOAD=""
 
 # Install Google Chrome and the corresponding version of ChromeDriver.
 ARG google_package_keyring
-ARG chromedriver_url=https://chromedriver.storage.googleapis.com/
+ARG chromedriver_url=https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/
+
 COPY --from=builder $google_package_keyring $google_package_keyring
 RUN arch=$(dpkg --print-architecture) && \
     echo "deb [arch=${arch} signed-by=${google_package_keyring}] https://dl.google.com/linux/chrome/deb/ stable main" \
         > /etc/apt/sources.list.d/google.list
 RUN install_packages dumb-init google-chrome-stable unzip
 # TODO: support arm64 when available (perhaps by just switching back to ChromiumDriver?).
-RUN chrome_ver=$(google-chrome --version | grep -Po '\d+\.\d+\.\d+') && \
-    chromedriver_ver=$(curl -LSfs "${chromedriver_url}LATEST_RELEASE_${chrome_ver}") && \
-    curl -LSfs "${chromedriver_url}${chromedriver_ver}/chromedriver_linux64.zip" \
-        | funzip >/usr/bin/chromedriver && \
-    chmod 755 /usr/bin/chromedriver
+RUN curl -o /tmp/chromedriver-linux64.zip "https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/115.0.5790.98/linux64/chromedriver-linux64.zip" && \
+        unzip -j /tmp/chromedriver-linux64.zip chromedriver-linux64/chromedriver && \
```

Comment:
(if you're keen to avoid the cruft now then I would just `rm -fr /tmp/*` instead of `/tmp/chromedriver-linux64.zip` on what's currently line 43 — but no big deal to leave as-is)

Reply:
Cool, I've updated that line.